ash-window/Cargo.toml

```diff
@@ -14,15 +14,15 @@ exclude = [".github/*"]
 workspace = ".."
 
 [dependencies]
-ash = { path = "../ash", version = "0.35", default-features = false }
+ash = { path = "../ash", version = ">=0.34, <=0.36", default-features = false }
```

Comment:
As discussed earlier I'd like to test-drive a range selection for `ash`, such that we don't have to perform as many breaking releases for `ash-window` when nothing else changes. I checked backwards compatibility and a change inflicted by `ash` last happened in 0.34. As such, this crate compiles fine with `ash=0.34` all the way up to `0.36`.

This was tested by removing `ash-window` out of the workspace temporarily (get some strange `rmeta` duplicate errors otherwise) and dropping the `path="../ash"` here to use the crates release.
Then I alternated the version of the example below (which is how users of `ash`/`ash-window` select their version of `ash`), which implicitly dictates the version that is used for `ash-window` as well (as long as it is within the allowed range, of course).

Now, when we release a new `ash` version that is compatible with `ash-window` without further changes, we can just perform a non-breaking patch release that extends the range with a newer version.

Sounds sensible/useful?

Reply:
:thinking: 
What is the usecase? I assume people can do `ash-window = "0.10"` and then they don't have to bump the version because we release patch version. I am not sure how useful this will be in practice.

> As discussed earlier I'd like to test-drive a range selection for ash

Can you share a link to the discussion?

Reply:
@MaikKlein You self-answered the question, it is indeed as I wrote above:

> such that we don't have to perform as many breaking releases for `ash-window` when nothing else changes

> Can you share a link to the discussion?

In hindsight I don't think we've truly discussed this, more of an off-hand mention that I wanted to try this some day. It might not even have been on this repo, perhaps on gitter or a different project entirely.

Reply:
That one-sided discussion might have been https://github.com/rust-windowing/android-ndk-rs/issues/211#issuecomment-1014707079.